src/pages/15Years/15Years.vue

```diff
@@ -1,33 +1,36 @@
 <template>
-	<www-page
-		:header-theme="headerTheme"
-		:footer-theme="footerTheme"
-	>
-		<div class="page-wrap">
-			<fifteen-years-styles>
-				<section>
-					<fifteen-years-header />
-				</section>
-				<section>
-					<fifteen-years-timeline />
-				</section>
-				<section>
-					<fifteen-years-individuals />
-				</section>
-				<section>
-					<fifteen-years-sponsors />
-				</section>
-				<section>
-					<fifteen-years-how-kiva-works />
-				</section>
-			</fifteen-years-styles>
-		</div>
-	</www-page>
+	<div>
+		<appeal-banner-15 :appeal-banner-content="{}" />
```

Comment:
I don't really understand how appeal-banner works, but i'm pretty sure this isn't where it should go.

Reply:
Correct. The global promo banner and any type of appeal is selected in the GlobalPromotionalBannerContentful.vue file. Selection is based on the first found Banner from the contentful setting AND the type of the Banner.

Reply:
So you can just remove it here as in will be included with www-page